README.rst

```diff
@@ -15,6 +15,8 @@ Django Admin lightweight date hierarchy
 Django Admin date_hierarchy with zero queries
 ----------------------------------------------
 
+**NOTE: You don't need this extension now; this issue has been fixed in Django. See: https://code.djangoproject.com/ticket/28933 **
```

Comment:
I think it's worth mentioning the Django version after which the functionality of this extension was included in Django, something like:

```md
**NOTE**: the functionality provided by this extension is [included as part of Django](https://github.com/django/django/pull/9469) starting at version 2.0. 
```

Reply:
Sure. Please check the new commit.